resources/config.yml

```diff
@@ -4,152 +4,151 @@
 # Otherwise the programm will crash.
 
 cmds:
```

Comment:
Although not really the scope of your ticket it would be great if you could update this config to be using the Meta key instead of the Alt key

THX